### PR TITLE
[WIP] Xenobio Skeleton Simplemob Rework

### PR DESCRIPTION
**Pull request by IndieanaJones**
## About The Pull Request

Ever make some gold slime core mobs as a xenobiologist?  Did you ever generate some really cool skeletons, only to realize there's no reason to make them sentient but to make them suffer, and there's no good drops to get from them either?  Well, today's PR solves that issue by removing the skeleton simplemobs (not the plasma miners) from the gold slime pool, and adding an entirely new new mob in its stead.

Now, instead of having reanimated skeletons and ice skeletons, both of which were useless when compared to the many other options a xenobiologist has, the gold cores now can spawn in a robust skeleton, who looks identical to the reanimated skeleton and has the same base stats as the reanimated skeleton as well, but contains new features to make the mob worthwhile.   Your robust skeleton comes with the following features:

- Even though you have that crappy health pool, damage, and speed stat inherited from your regular counterpart, death doesn't get you down easy.  Rather, you'll find yourself getting up only a short while after death!  Unless gibbed, you will still spawn remains on death, but revive from those remains after a short time has passed!  However, if your remains are cleaned up or otherwise removed, you'll find that your death is quite permanent.

- So, now you have your skeleton band, and life doesn't suck so much anyone now that killing you takes a bit of elbow work, but you're still a bunch of crappy simple mobs at the end of the day, right?  Well, while your simple mob status might not be fixable, you'll find you're not JUST some crappy simple mob.  If 5 skeletons can come together, they can merge and form a GRAVE WARDEN.  The Grave Warden is a larger  being made out of skeletons, and all the skeletons a part of the mix get to control it!  It's pretty resistant to damage but not fast, and can damage doors (but not broken airlocks) by colliding into them. 

 What makes the Grave Warden special is that colliding into a humanoid species tears out their skeleton and adds them to the Grave Warden as well, healing it slightly, and since they are now a skeleton, also gives them shared control.  Other mobs (excluding megafauna) also get added to the Grave Warden on collision, but cannot control it.  When the Grave Warden is killed, it explodes akin to a clown car and releases all the mobs and skeletons inside.  Mind you, skeletons and mobs can get out of the Grave Warden akin to getting out of a clown car as well, so if the Grave Warden needs to bust in somewhere, it'll need to get everyone to stop and unload the troops, but that likely won't be easy.  Hectic skeleton fun for everyone!

## Why It's Good For The Game

Right now, compared to what else xenobiologists have to make sentient, there's no reason to make a skeleton sentient at all other than short-lived memes.  With this change, there's an actual reason to make sentient skeletons for both crew (bodyguard who requires extra steps to kill) and traitors (unleashing a grave warden onto the unsuspecting crew).

## Changelog
:cl:
tweak: Xenobiologists have noted that captive gold spawn skeletons having been putting their heads together recently...
add: Fear the Grave Warden!
/:cl:

EDIT: New feature considered: skeletons having hands.  Please discuss your thoughts below.

### CODE REVIEW

**Comment by anconfuzedrock** [comment]
literally zombies but with cheerleading

**Comment by necromanceranne** [comment]
Rock ain't wrong. This seems like xenobio getting romerol for free but without the hitch of having the zombies turn on you. And also makes a megafauna because ???

Frankly this seems real funny if it wasn't a xenobio thing.

**Comment by Shadowflame909** [comment]
I hope the grave warden is megafauna size for extra spook

**Comment by wesoda25** [comment]
Xenomobs who can reproduce are absolute fucking cancer, just remove that portion and it’d be neat

**Review by Tlaltecuhtli — APPROVED** [review]
looks fair and balanced

**Comment by tralezab** [comment]
Since the grave warden is a vehicle instead of a mob (that is so funny for some reason) maybe you should have the grave warden be a big ass fuckin ball of skeletons like in dark souls

<details>
  <summary>WARNING: Spooks inside!</summary>

  ---

image


  ---
</details>

File: code/modules/mob/living/simple_animal/hostile/skeleton.dm [hunk 1]
```diff
@@ -122,3 +121,163 @@
...
+	newskele.faction = faction.Copy()
+	if(L.mind)
+		L.mind.transfer_to(newskele)
+	L.gib()
+
+/obj/vehicle/sealed/car/grave_warden
+	name = "grave warden"
+	desc = "A pile of skeletons somehow forming one large skeleton."
+	icon = 'icons/mob/lavaland/lavaland_monsters.dmi'
+	icon_state = "legion"
```
Comment (tralezab) [inline code/modules/mob/living/simple_animal/hostile/skeleton.dm]: you don't have to do my suggestion of a skeleton ball but reusing the sprite of a common lavaland mob is a really bad idea

**Comment by Ratvar** [comment]
Ball of Fun would be so cool if it was a changeling horrorform ability instead, but if it's added as a skeleton, it won't ever be copied over
And yes, this is real strong. What weakness xenobio slave zombies have? I think that soap/cleaning tools are *less* available then spears to cut off someone's head.

**Comment by RogueSteampunker** [comment]
I mean, maybe make it to where you can click the remains to slowly part them away? That way it isn't as if a single skeleton is impossible to put down, but makes a group of them a much more dangerous foe as they protect their regenerating brethren.

**Comment by datorangebottle** [comment]
No. Gold slime does not need buffs, it's strong enough already. Xenobio already has three different armies it can make- golems(which are underused), slime army, and gold slime mob army. It doesn't need a fucking fourth.

**Comment by Shadowflame909** [comment]
Well. Gold Slimes did lose Headslugs. This would be a neat replacement without the conversion element

**Comment by datorangebottle** [comment]
> Well. Gold Slimes did lose Headslugs. This would be a neat replacement without the conversion element

They lost headslugs because they were already super fucking broken and gold slimes still are. Magicarp and chaos carp are disgustingly powerful, and then there's "horde of fucking spiders" tier.

**Comment by Shadowflame909** [comment]
To be quite honest if you remove magicarps and time stop from Xenobiology. It's the same level as Botany

**Comment by datorangebottle** [comment]
> To be quite honest if you remove magicarps and time stop from Xenobiology. It's the same level as Botany

I never said remove time stop. I said remove magicarps and spiders. Or rework gold slime altogether, because it doesn't fit what it should be. Honestly, time stop and golems should be the gatekeeper. If a mechanic is stronger than those on paper, it shouldn't be in the game as a nonantagonist, especially not for science as a whole.

**Comment by Shadowflame909** [comment]
You can take my immortal statue, you can take my headslugs, you can even take spiders because no one likes them anyways.  Time stop is also pretty lame


But Staff of Change Magicarp will always be the holy grail for xenobiology. 

At most, I believe that they should only spawn when you use plasma. So you have to worry about an angry carp firing death-bolts at you.

But, as Kor once said when they were still here. "You can't just take the holygrail of a department away." Because it'll lose the thing that keeps people going.


And Crossbreeding isn't all that powerful either intentionally. It's pretty gimmicky and slightly useful.


But lastly, if you take away literally every type of chaotic method Xenobiologists can use like Time Stop, Golems, Black Slime Mutation Toxin. But leave Magicarp, people will still want to do the job.

But if you do the inverse, it'll mellow out like the experimentor.


Since we're getting off topic. I think this Skeleton Simplemob is a perfect fit for Xenobiology, but it needs some slight tweaks. 

Slow Golems and Slow Simple Mobs are a joke and are easily taken care of. Sentient spiders are hellish, but no one likes them and only a select 4 static names even bother with using them.


That tree simple mob with the ability to stun is pretty scary though, that's it. So in conclusion, this PR isn't a game changer, but it'll be fun to see used for 3 rounds or so.

**Comment by datorangebottle** [comment]
> But Staff of Change Magicarp will always be the holy grail for xenobiology.
Staff of change carp is literally one of the most cancerous incidences. It devolves the round into changed people fucking with unchanged people and vice versa.

> At most, I believe that they should only spawn when you use plasma. So you have to worry about an angry carp firing death-bolts at you.
You might as well remove them at that point, because then you can't make them sentient without them killing you unless they're not res or door creation.

> But, as Kor once said when they were still here. "You can't just take the holygrail of a department away." Because it'll lose the thing that keeps people going.
Just because you think it's the holy grail doesn't mean it is the holy grail. They still have a ton of cool things to do without fucking the station over with mobs, which is currently the only thing they do because it's so much more powerful than the other options.

> And Crossbreeding isn't all that powerful either intentionally. It's pretty gimmicky and slightly useful.
I never said remove crossbreeding. I really like crossbreeding.
> But lastly, if you take away literally every type of chaotic method Xenobiologists can use like Time Stop, Golems, Black Slime Mutation Toxin. But leave Magicarp, people will still want to do the job.
I didn't say take away everything they have to cause chaos. Stop moving the goalposts.

>That tree simple mob with the ability to stun is pretty scary though, that's it. So in conclusion, this PR isn't a game changer, but it'll be fun to see used for 3 rounds or so.
You're right, it won't change anything besides keeping Gold Slime as the current king of xenobio and being a reason to prohibit its much-needed rework or removal.

**Comment by IndieanaJones** [comment]
Alright, seeing as the skeletons would be a tad too strong with revive mechanics and the ability to make more skeletons, I've removed that ability from their toolkit.  This also means to create the Grave Warden, the xenobiologist will need to get 5 skeletons from the gold mob roulette, making it a great deal harder of a tool to obtain.  I have, however, left the conversion ability of the Grave Warden because in the time it takes to make 5 sentient skeletons, *someone* should have noticed what the xenobiologist is up to.

However, to cover everyone's comments here:
- They can't reproduce now, so the comments about them being better zombies + being reproductive simples are now void.  The skeletons, also unlike zombies, do not slowly kill you with just one touch, and also have terrible stats compared to zombies, being slow, weak, and having little health.  Considering now their reproductive ability is removed, I think having to clean up their remains taking some elbow work balances itself out pretty well, because they're won't be so many unless you let the xenobiologist make a Grave Warden.

- The Grave Warden is not a megafauna, despite it's appearance.  It's actually just a slower, tankier clown car made to look like a big monster.  This means it has no attacks or anything of the sort, just it's collision-based abilities.  To be honest, I expect the Grave Warden will barely be a threat, because you'll have 5+ people trying to control it at once, and I doubt that will work out most of the time.  It's current appearance as a large legion mob honestly fits, though I also love DS3 and the skeleball, but I am no spriter, which sucks in terms of being a limiter.  I'll see what I can do.

>datorangebottle talking anything about xenobio

That's gonna be a yikes from me, dog, though I will respond to parts of your commentary, mainly about the role of gold slime cores in xenobio.  Honestly, the reason I use them so often?  Ghost roles.  All the current normal ghost roles suck and and outside of being picked to be an antag, which is fun, the other normal ghost roles are heavily disassociated with the main round and usually involve very boring and repetitive gameplay.  This stems from the fact most of them just act as tutorials for different mechanics.  With golds, you actually get some fun RP and get to do things associated with the station and roun.  Though most of the mobs are a novelty akin to the other ghost roles. they're usually pretty fun for the people that take them, and I certainly get takers often enough.

Golems also exist, but I've noted in my time that few people actually take golems over being a mob, despite most people agreeing that golems are more powerful overall (because they have hands).  The only exception I've noted is cardboard golems, mainly from their unique niche of replicating all over the station and having their own little gangs, it's quite enjoyable.  But, if you were to take golds from xenobio, what you're left with is a station which just focuses on self-powergaming, and nothing else.  While there are people who already play xenobio like this, people like myself play it to give others a good time.  Sentient mobs can also be very helpful, as a lightgeist, revive carp, or some of the physically stronger mobs can be tide turners when handling certain antags.  It also adds to the crazy theme of SS13, a setting where the impossible is most certainly possible.

Also, slightly off-topic, but slime army isn't even worth a second thought anymore.  Slimes basically lose all movement speed upon being slightly chilled and have no answers to handling cyborgs, so they really aren't a main option anymore as they used to be.  Golem army also usually just involves having that one robust guy become a golem and fuck up atmos or build a mech, so it really doesn't involve as many people as I'd personally like.  As for death bolt carps, they're pretty rare, and it usually takes RNG or a good understanding of xenobio to grind them out fast enough, and crossbreeding is so self-focused that I don't usually focus it.

File: code/modules/mob/living/simple_animal/hostile/skeleton.dm (hunk 1 above)
Comment (IndieanaJones) [reply code/modules/mob/living/simple_animal/hostile/skeleton.dm]: Considering it's sized up, it's pretty obvious it's not a legion.  As I cannot make sprites, I'll have to figure out something.  I really like the DS3 skeletonball though, so if I can, I'd want that.

**Comment by datorangebottle** [comment]
> > datorangebottle talking anything about xenobio
> 
> That's gonna be a yikes from me, dog, though I will respond to parts of your commentary, mainly about the role of gold slime cores in xenobio.

Wow, way to contribute to discussion. I take offense to that, I have quite a bit of experience with xenobiology and you can't just throw my opinion out because you disagree with it. The department as it is is extremely overpopulated with things to do and is far less linear than other departments- they shouldn't have need for absolutely broken options.

>first paragraph

Granted, some of the gold slime roles are really novel, but a good portion of them are 'you can smack things and that's about it'. A much smaller portion have actually different gameplay, and even smaller than that are the ones that have different gameplay without being hideously broken. Golems are only more powerful than most options once armed, and they tend to be ignored completely for other things like posibrains or ghost roles. Existing ghost roles being shit doesn't make an excuse for one that is actively harmful to the ongoing round in several ways- magicarp in any form cause absolute chaos and fuck with antags, basic mobs are almost exclusively only able to help via combat and breaking shit- the only one that doesn't is the lightgeist, as far as I'm aware.

> second paragraph

Golems should be taken more and I'm honestly not sure how they aren't, being one of the most unique roles on the station outside of gold slime mobs. Outside of piloting mechs, however, golems don't beat out the pushed options of the gold slime, which is generally why it's used more. 

The argument about self-powergaming is invalid because we already have other stations that perform that way- genetics and robotics, for a couple. There is nothing stopping either of those stations from handing out their 'self'-powergaming tools, and both of them often do, only sometimes to the detriment to the station when they're irresponsible with said powers.

Finally, magicarp are completely way too pushed. Headslugs were removed because getting the powers of an antagonist as a non-antagonist is fucking stupid, and magicarp are the same sort of thing but better. Imagine being able to buy a random staff that could pilot itself, with somewhat low hp. Fast-moving, spaceproof, don't need to breathe air, and I believe they also have a melee attack? All types of magicarp are too powerful, with the least powerful being all-access. In order of power:

-Door Creation: all-access, which isn't nearly as powerful as the other options. They represent infinite materials for science, as well as leaving massive, randomly-colored easy-access eyesores all over the place. Kind of just a pain in the neck.
-Chaos: Has a chance of all the other carps, but at random, making it less reliable and thus powerful than the rest of the carp options.
-Resurrection: fucks over antagonists, because you can revive any corpse that you can find. Also helps antagonists when it inevitably, mistakenly, revives someone who was or starts murderboning. Makes death a non-consequence, which is something the codebase seems to be moving away from.
-Death Bolt: Instantly kill anything that isn't inside a container or mech. Super easy to see why it isn't fun for anyone who isn't the carp or its master. A death carp doesn't have to be sufficiently robust- they just have to hit their target once.
-Change: Transform any human or cyborg or whatever into another thing. Strips the person you transform, allowing you to steal things like all-access, tools, or weapons. I'm pretty sure it also removes implants?  Allows syndicate borgs to appear on the station during non-nukie rounds, xenomorphs without a xeno spawn event, and monkeys if you really wanted to fuck someone over. Also, staff of change bolts are currently bugged so that they randomly make people just fucking vanish, removing them from the round permanently. When a staff of change carp is on the station, the round devolves into absolute fucking chaos.

> Also, slightly off-topic, but slime army isn't even worth a second thought anymore. Slimes basically lose all movement speed upon being slightly chilled and have no answers to handling cyborgs, so they really aren't a main option anymore as they used to be. Golem army also usually just involves having that one robust guy become a golem and fuck up atmos or build a mech, so it really doesn't involve as many people as I'd personally like. As for death bolt carps, they're pretty rare, and it usually takes RNG or a good understanding of xenobio to grind them out fast enough, and crossbreeding is so self-focused that I don't usually focus it.

Slimes do have advantages despite having an easy answer all over the station. They're easy to mass produce, have stuns in adult form, and can make an army of nonsentient slimes litter the station. It'd be pretty fun if anyone ever did it. I take sentient slime whenever it shows up due to the RP options, mostly. It can be pretty entertaining to just sit there and blorble uselessly all round. 

Golem army's problem is usually getting armed, somehow- other than that, they're a perfectly viable option that just isn't used or taken enough. I've used them in a traitor round recently to good effect, even though only two showed up at any given time. I made plasma golem shells and had them run around for a bit, then made an adamantine and had it order them to cause chaos.  

All the magicarps are annoying to have in a round at best, for reasons i've stated above.

Crossbreeding is actually what I usually focus on during my xenobio rounds- I enjoy them and I wish the list were complete, even if it would mean some completely fucking broken options.

**Comment by datorangebottle** [comment]
Adding something to that, but using a separate post so that I don't edit things in to a post you may have quoted already-


Adding more options to gold slimes when they already turn xenobiologists into a wizard is really not something we need.

**Comment by optimumtact** [comment]
This is a stupid ass change

**Comment by IndieanaJones** [comment]
> first response

I disagree with gold core mobs being harmful to the round, as long as the player behind the mob isn't a total dickweed, it gives the ghost a way to interact with the round and acts as a product of the xenobiologist's work, just as a cyborg is the work of a roboticist, which usually isn't harmful, unless again, the player is a dickweed.  Personally, I would like it if more xenobio mobs had applications like the lightgeist does, a helpful utility which doesn't involve unga bunga, me left click.  In hindsight, this PR doesn't server that goal, but just adds an antag utility to skeletons and a QOL ability as well.  A solution I'd think of to this would actually be rather simple, just let the skeletons use their hands.

> second response

Ever hear of "You can lead a horse to water, but can't make it drink"?  That's golems.  They are overall better than most xenobio mobs, but I see your concern of how they are overshadowed by carps, which can be the be-all-end-all of xenobio.  Considering how the rest of xenobio is, I think it'd be an awful idea to remove carps from the pool until crossbreeding is done and we can see if we have something to fill it's void.  I personally would rather buff and make unique the rest of the xenobio mobs and then think about removing carps.

> third response

Slimes barely aggro on humans anymore ever since their tweak.  Sure, they get around easy and it's simple to populate the station with them, but a borg with a fire hydrant can kill infinite amounts of them with no challenge.  Believe me, I've attempted slime invasions before, but they never get the results needed to be considered viable.  The sentient ones get chilled and slowed, which effectively kills them from one shot, and the non-sentient ones pose no threat at all.  I wish they did, because I would definitely use them in that case.

And, about magicarps being annoying, it seems very subjective.  A lot of people like them, a lot of people don't.  Seems like something we should have a server vote on, spiders too.  And, about the initial comment, I will apoligize, shouldn't have made it.  Any time xenobio is ever mentioned, you're usually a very aggressive anti- everything I usually stand for, and instead of trying to discard your opinion, we should debate and reach a compromise as we should now.

**Comment by IndieanaJones** [comment]
Also, is this change necessary?  No.  Does it harm the balance of the force?  I also disagree.  I think as long as it isn't broken, it's fair game to give it to them.  Skeletons exist as an option now, just a useless one.  Like I said, I'd like to diversify and give the other xenobio mobs some helpful utilities so there's more depth to them than "I attack things :)".  This doesn't really go with that line of thinking, but if it's the popular idea, we can change this PR into something else.  I personally think if we give them the ability to use hands, it would make them a nice, rounded mob.  Hands are very useful and allow them to do a good deal of things other mobs can't, the revive on death gives them a niche, and the skeleball/ Grave Warden gives them the ability to prove a good antag utility if enough work is put in.

**Comment by 4dplanner** [comment]
If you're removing the conversion aspect, you should edit the OP

**Comment by IndieanaJones** [comment]
Done.  Thanks for the reminder, friendo.

**Comment by Shadowflame909** [comment]
Does the Skeleball/grave warden have Gas Protection?


If not, then I can think of a pretty cool way to force people to run into a smoke machine filled with slime mutation toxin, keeping them there until they turn into slimes.


If so, then I guess it'd be a cool way to hijack by forcing people out of airlocks

**Comment by IndieanaJones** [comment]
Since the skeleball/Grave Warden still turns people into skeletons, so basically anything in the ball gets gas protection, yeah.

**Comment by datorangebottle** [comment]
> 1

The gold core mobs are almost completely harmful to the round as they are. Carps ruin rounds for a few people at best(by resurrecting targets or killing antags, or vice versa), and for everyone at worst(killing everyone, breaching the hull, giving all-access to everyone, staff of change). Spiders also ruin the round for everyone but the spiders. Most of the mobs can only robust things- they don't even have a help intent. 

I think the skeletons being more similar to pirate / wizard skeletons would be fine, i.e. being able to use their hands and being spaceproof, not having to breathe, but they're like more fragile humans. The skeleton ball is a cool idea, but giving them access to a free traitor item(the clown car) is pushed- they already have problems with making themselves into antags that need to be adressed. Yes, gold slime mobs should all be various forms of helpful- the shambling miner should be able to, yknow, MINE, after all. If someone doesn't want to work on that, then gold slimes should just be completely reworked to be more in line with metal/adamantine slimes. An idea that was tossed around on discord was that the gold slime could be related to wealth- with a minor activation making a random amount of cash money, and a major activation making random mineral coins, which can be deconstructed by science to use as minerals when mining isn't present, but is still a bit of a hassle to do.

> 2

I don't think adding more features should be done at all until the gold slime pool is balanced with other options. The station shouldn't have access to sentient wizard spells. They outright should not. Wizard is overpowered because it's one man versus the station. Magicarp are overpowered because they're miniature wizards. The mob isn't even used in normal events, generally, just when an admin hits a button, because they're very fucking hard to deal with even when they aren't controlled by a bad AI. Either they're door creation or res carp, and thus mostly harmless to actual people (despite having massive consequences for the round as a whole) or they're staff of change or death bolt carp and just completely derail the round.

Making the other mobs viable is a cool idea, again, but magicarp should just flat-out be removed from the pool if you want to keep gold slime at all near its current state.

> 3

Slimes can still be ordered around by the xenobiologist or his minions. Sure, slimes can turn out really badly- but you're in science and should be able to handle a cyborg if it's killing your minions. You have access, generally, to the RD console or a flash. You can order them to hold still for a moment if they don't know you caused the slime invasion, and remove their power cell. You're ignoring that you can actively assist the slime army. Yes, it is a very weak option. But it's an option all the same, and it's rarely if ever used. Even when I try, nobody signs up for sentient slime, and it's sad to see. I think the slimes themselves need a buff to be slightly more aggressive and slightly harder to kill.

> And, about magicarps being annoying, it seems very subjective. A lot of people like them, a lot of people don't. Seems like something we should have a server vote on, spiders too. And, about the initial comment, I will apoligize, shouldn't have made it. Any time xenobio is ever mentioned, you're usually a very aggressive anti- everything I usually stand for, and instead of trying to discard your opinion, we should debate and reach a compromise as we should now.

I do think a server vote is warranted on this feature as well. I totally agree with you on that.

I've slowly become more and more against xenobiology getting new features. I used to play it a lot, and found that whenever I wasn't playing it, the gold slime was the big thing that every xenobiologist did. I got increasingly jaded by this- the fact that a department with such a wide swathe of options only really uses one- that I came to hate the department quite a bit. So yeah, I am aggressively anti-xenobiology for that reason. Thank you for apologizing, though.

**Comment by IndieanaJones** [comment]
Well, for starters, most people are likely going to straight up disagree with the gold slime rework.  A lot of people like it as it is right now, just look a the votes on this post.  If you think gold slime cores being reworked to be boring is what's actually wanted by a majority, the general gist I get it is that the majority likes it the way it is, but a vote would definitely straighten that out.  As I also stated, I want to look into reworking the simple mobs to have utility tasks as well, so at the very least, I think compromise could be found there.

As I stated earlier, a vote would solve the magicarp (and spider) issue.  If the majority wants it, it stays, majority dislikes it, remove it.

The slimes' ordering mechanic is bugged, and sometimes just fails to affect them, despite being a friend/slimeperson.  I think they really need a buff.

I've been playing xenobio for a long-ass time as well, and back in the day I loved it for the golem aspect.  The mobs made it a lot more enjoyable when they got added, though I think they made the job a bit too easy for what it gives with the console and whatnot, but I'm a minority opinion on that.  I get jaded when I see xenobiologists make no shells and no sentients considering it's what makes xenobio as unique as it is in my opinion, but then there's also people like you who feel differently, so it is what it is.  Also, I am Charles Dunley, so you can have your confirmation for ragging on me on Discord :)

File: code/modules/mob/living/simple_animal/hostile/skeleton.dm (hunk 1 above)
Comment (IndieanaJones) [reply code/modules/mob/living/simple_animal/hostile/skeleton.dm]: Also, it plays Megalovania sounds when it moves.

**Comment by 4dplanner** [comment]
Votes are a bad way to design a game. On topic, the new version of this doesn't seem too bad, honestly.

**Comment by ATH1909** [comment]
> You might as well remove them at that point, because then you can't make them sentient without them killing you unless they're not res or door creation.
Friendly reminder that the chaplain's null rod renders you immune to magicarp bolts.

**Comment by 4dplanner** [comment]
You don't need nullrod, xenobio has the tools entirely from slimes

**Comment by Shadowflame909** [comment]
You don't need xenobiology holymelons make you immune to cult stuns and magic

**Comment by FantasticFwoosh** [comment]
Im not sure big paragraphs were required to properly elecute people's feelings on chattering boney-fools driving through the station headbanging in a skeleton megazord and playing a smash-mix of "what is love" full blast.

I hope to liberate the skeletons from inside the crew for the immediate future.

**Comment by ExcessiveUseOfCobblestone** [comment]
lol xenobio in a nutshell

- Is the PR a buff: justify it using crossbreeding not being in the game yet
- Is the PR a nerf: oppose it using crossbreeding not being in the game yet

**Comment by Potato-Masher** [comment]
This is fun. I'm a fan of fun. Let's do it, skeletons merging into a massive ass skeleton thing sounds hilarious if a xenobiologist can pull it off.

**Comment by optimumtact** [comment]
this is silly and more suited for a wizard type deal

**Pull request closed without merging**